libcnb-test/src/pack.rs

```diff
@@ -76,6 +76,9 @@ impl From<PackBuildCommand> for Command {
             pack_build_command.builder,
             String::from("--path"),
             pack_build_command.path.to_string_lossy().to_string(),
+            // Adjust pull-policy to prevent redundant image-pulling, which slows CI and risks hitting registry rate limits.
+            String::from("--pull-policy"),
+            String::from("if-not-present"),
```

Comment:
Nit-ish: `PackBuildCommand` was meant to abstract away the building of the `pack` command from out actual logic which flags should be passed, etc.

I would love to see a `pull_policy` field on `PackBuildCommand` that is of type `PullPolicy` and the "logic" (even though it a static value at this point) moved to the call-site to keep the separation of concerns.